src/Uno.UI.Tests/RoutedEventTests/Given_TappedRoutedEvent.cs

```diff
@@ -81,6 +81,27 @@ public void When_SubscribingUsingAddHandler_And_Unregister()
 			events.Should().HaveCount(1);
 		}
 
+		[TestMethod]
+		public void When_SubscribingUsingAddHandlerTwice_And_Unregister()
+		{
+			var events = new List<(object sender, TappedRoutedEventArgs args)>();
+
+			var root = new Border();
+
+			void OnTapped(object snd, TappedRoutedEventArgs evt) => events.Add((snd, evt));
+
+			root.AddHandler(UIElement.TappedEvent, (TappedEventHandler)OnTapped, false);
+
+			root.AddHandler(UIElement.TappedEvent, (TappedEventHandler)OnTapped, false);
+
+			root.RemoveHandler(UIElement.TappedEvent, (TappedEventHandler)OnTapped);
```

Comment:
That's a very strange scenario: adding twice the **SAME** delegate.

Reply:
It's unusual to do deliberately, but it occurs fairly easily with code like the following:
view.PointerPressed += OnPointerPressed; // 1 => 2
_serialDisposable.Disposable = Disposable.Create(() => { //.Disposable is non-null, 2 => 1
    view.PointerPressed -= OnPointerPressed;
    ....